From a6dfcd03425656208fc548c75f955016c26e8f71 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 21 Sep 2007 19:49:04 +0000
Subject: [PATCH] Jie Zhang writes:
 GCC 4.3 will put fixed headers in a seperate include-fixed directory.
 And I don't think that there are people who are still using the 'stupid'
 GCC which cannot handle -iwithprefix, at least it works with gcc-3.3.x.

---
 Rules.mak |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/Rules.mak b/Rules.mak
index a8655d2..1cfffc3 100644
--- a/Rules.mak
+++ b/Rules.mak
@@ -309,9 +309,7 @@ ifeq ($(UCLIBC_BUILD_NOW),y)
 LDFLAGS+=-z now
 endif
 
-# Sigh, some stupid versions of gcc can't seem to cope with '-iwithprefix include'
-#CFLAGS+=-iwithprefix include
-CFLAGS+=-isystem $(shell $(CC) -print-file-name=include)
+CFLAGS += -iwithprefix include-fixed -iwithprefix include
 
 ifneq ($(DOASSERTS),y)
     CFLAGS += -DNDEBUG
-- 
1.6.4

